packages/SwingSet/src/kernel/vatManager/manager-helper.js

```diff
@@ -170,8 +170,8 @@ function makeManagerKit(
     transcriptManager.startReplayDelivery(expectedSyscalls);
 
     // we slog the replay just like the original, but some fields are missing
-    const newCrankNum = undefined; // TODO think of a way to correlate this
-    const kd = undefined;
+    const newCrankNum = /** @type {any} */ (undefined); // TODO think of a way to correlate this
+    const kd = /** @type {any} */ (undefined);
```

Comment:
I think we want to avoid inline jsdoc. 
```suggestion
    /** @type {any} */ 
    const newCrankNum = undefined; // TODO think of a way to correlate this
    const kd = undefined;
```
That'll also leave the git blame untouched

Reply:
FWIW `newCrankNum` in the slog is usually a BigInt, but in this case we don't know what it is, so we give the slogger `undefined`. Likewise, `kd` is usually a `KernelDeliveryObject` (defined in types.js), but during replay we don't have the kernel version, so we can't give one to the slogger.

Reply:
Ah, so…
```suggestion
    /** @type {bigint=} */ 
    const newCrankNum = undefined; // TODO think of a way to correlate this
    /** @type {KernelDeliveryObject=} */ 
    const kd = undefined;
```

(in .ts w'd say `bigint?` for optional but TIL that JSDoc form interprets that as `bigint | null`)

Reply:
so I switched to above instead of inline. However I had to keep `any` because the function does not accept `undefined` in those places. It seems more like a hack that `undefined` plumbs through without throwing anything, but I don't want to change that behavior.